Fluid.MvcViewEngine/FluidRendering.cs

```diff
@@ -62,6 +62,9 @@ static FluidRendering()
             context.AmbientValues[ViewPath] = path;
             context.AmbientValues.Add("Sections", new Dictionary<string, IList<Statement>>());
 
+            var viewsPath = Path.Combine(_hostingEnvironment.ContentRootPath, "Views");
```

Comment:
And the `Shared` folder could be an option too with it by default, that would totally make sense.